yarn.lock

```diff
@@ -25,6 +25,10 @@ abbrev@1:
   version "1.1.0"
   resolved "https://registry.yarnpkg.com/abbrev/-/abbrev-1.1.0.tgz#d0554c2256636e2f56e7c2e5ad183f859428d81f"
 
+abbrev@^1.0.7:
```

Comment:
This file is huge and you have say nothing about that in your PR description.

Reply:
It is generated automatically by yarn when you install/update dependencies.